src/pavan/Assignment17/MaxLengthArray.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *Program 8: return the name having maximum length.
+input : {"Maulik", "Krishna", "Technocredits", "Kangan", "Harsh"};
+output : Technocredits
+ */
+
+package pavan.Assignment17;
+
+public class MaxLengthArray {
+	
+	String  maxLength(String[] args) {
+		int lengthofnm = args[0].length();
+		String maxstring= null;
+		
+		for (int index = 1; index < 5; index++) {
```

Comment:
you should use args.length not hard-coded value, in future array will 10 length then again you have to change that vlaue